eks/inputs.tf

```diff
@@ -61,3 +62,30 @@ variable "nodes_blue_min_size" {
 variable "nodes_blue_max_size" {
   default = 1
 }
+
+//the following below are required for setting up the vpn
+variable "foxpass_api_key" {
+  type = "string"
+}
+
+//use this for psk generation https://cloud.google.com/vpn/docs/how-to/generating-pre-shared-key
+variable "vpn_psk" {
+  type = "string"
+  default = "(;v&zeTM8#^QLH4Eb]]8Mo(B"
+}
+variable "vpn_group_requirements" {
+  type = list(string)
+  default = []
+}
+
+// these should be enabled later
+//variable "duo_skey" {
+//  type = "string"
+//}
+//variable "duo_ikey" {
+//  type = "string"
+//}
+//
+//variable "duo_api_host" {
```

Comment:
Do we need these? Don't they happen within Foxpass so they don't matter?

Reply:
no they are needed as part of the config.json if 2FA is enabled. AFAIK theres not way to use them from foxpass itself.